Sources/SwiftIfConfig/BuildConfiguration.swift

```diff
@@ -276,3 +291,11 @@ public protocol BuildConfiguration {
   /// #endif
   var compilerVersion: VersionTuple { get }
 }
+
+/// Default implementation of BuildConfiguration, to avoid a revlock with the
+/// swift repo.
+extension BuildConfiguration {
+  func isActiveTargetObjectFileFormat(name: String) throws -> Bool {
+    return false
```

Comment:
I think this method should throw an error that has a description like `'_objectFileFormat' is an experimental feature`. That way `_objectFileFormat` will not be allowed unless `BuildConfiguration` actively implements it. This way we also don’t silently assume that there is no active target object file format, as we do right now.

The `BuildConfiguration` implementation in ASTGen can then check whether the `_objectFileFormat` experimental feature is enabled. If not, it can throw a similar error. And if the feature is enabled, it can return `true` or `false`.

Reply:
Done